### PR TITLE
Continue and refine readme tasks

### PR DESCRIPTION
**Pull request by olh2012**
完善 CARLA 数据采集、闭环评估、多任务头和 RL 微调原型，以推进 README 中进行中及计划中的任务。

- 扩写 `scripts/collect_carla_data.py`，支持同步模式采集 CARLA 轨迹并提供离线伪数据 fallback；同时强化 `evaluation/closed_loop_sim.py`，可直接加载 `VLADrivingPolicy`、统计碰撞/越线/路线完成度并在无 CARLA 时退化为 mock 评估流程。
- 在 `models/policy.py` 中新增多任务头基础设施，更新 `examples/test_policy.py` 与 `configs/policy_config.yaml` 展示导航分类/车道偏移/避障得分的推理流程；文档同步说明多任务用法、采集脚本、强化学习原型与发布工具。
- 新增 `training/rl_env.py` 与 `scripts/rl_finetune.py` 构建轻量 RL 环境和 PPO 管线骨架，以推进“强化学习微调”计划；提供 `scripts/export_policy.py` 便于将 checkpoint 打包发布。

---
<a href="https://cursor.com/background-agent?bcId=bc-874eff12-ac3c-46c9-ba00-89987df27a4e"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/open-in-cursor-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/open-in-cursor-light.svg">Open in Cursor</picture></a>&nbsp;<a href="https://cursor.com/agents?id=bc-874eff12-ac3c-46c9-ba00-89987df27a4e"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/open-in-web-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/open-in-web-light.svg">Open in Web</picture></a>

---

> [!NOTE]
> Introduce multi-task prediction in VLADrivingPolicy, a CARLA data collector with offline fallback, revamped closed-loop evaluator, lightweight RL finetuning env/script, export script, and corresponding config/docs/examples updates.
> 
> - **Model (VLADrivingPolicy)**:
>   - Add configurable multi-task heads via `MultiTaskHead` (e.g., `navigation_logits`, `lane_offset`, `obstacle_score`) and labels; fuse outputs in `forward`.
>   - Extend `predict_trajectory(image_tensors, text_instructions, return_aux)` to optionally return aux outputs.
> - **Evaluation**:
>   - Rewrite `evaluation/closed_loop_sim.py` to load `VLADrivingPolicy`, run synchronous CARLA loop (or offline mock), apply controls from predicted trajectory, and report metrics; add CLI (`--town`, `--fps`, `--num-episodes`, `--max-steps`).
> - **Data Collection**:
>   - Add `scripts/collect_carla_data.py` with synchronous CARLA recording and offline pseudo-data; outputs `images/` and `annotations.json` per `data/DATA_FORMAT.txt`.
> - **Reinforcement Learning**:
>   - Add lightweight env `training/rl_env.py` (gymnasium-compatible) and `scripts/rl_finetune.py` (PPO skeleton) for RL finetuning prototype.
> - **Export Tooling**:
>   - Add `scripts/export_policy.py` to package checkpoints into `policy_state.pt` with `metadata.json` and copy config; supports `--mock`.
> - **Configs & Examples**:
>   - Update `configs/policy_config.yaml` to include `model.multi_task` (classes, tasks).
>   - Update `examples/test_policy.py` to consume `multi_task` config and print aux predictions via `return_aux`.
> - **Docs (README)**:
>   - Document multi-task outputs, data collection usage, closed-loop eval options, RL finetuning prototype, and export workflow; update roadmap status.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 6305d026df2107e0c3190a77ebee89888e045a05. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

**Review by olh2012 — COMMENTED** [review]
done
